### ISSUE TITLE
Send copy of User Messaging message to yourself

### ISSUE THREAD

**Issue by soytakyon** [SoylentNews/rehash#409]
Option or default to send a copy of a user message an editor sends to the editor's own messages. This way they can verify it was sent, and have a temporary copy of what they said. Sending it to the editor's email account could be another useful option.

You could prevent this copy from being sent in the (rare?) case where an editor goes on their own user page and sends themselves a message.

**Comment by TheMightyBuzzard** [comment]
Yeah, that should be pretty doable. Shouldn't be but one or two lines of code. Default would be better I think.

**Comment by soytakyon** [comment]
Come to think of it, might as well do the same thing for deletion reasons as well, which are just a specific form of editor-user messaging.

**Comment by TheMightyBuzzard** [comment]
Actually, they're an entirely different category but that's neither here nor there. I think this one goes to the spring update rather than the quick-fast December update.
